docs/workshops/overview.md

```diff
@@ -1,4 +1,9 @@
 # Arcadia Workshops
 
 This is the landing page for workshops hosted at Arcadia Science. 
-No workshops have been hosted by Arcadia yet, check back soon for materials.
+Workshops are longer hands-on trainings that focus on building specific skill sets.
+See below for lesson materials for workshops that have been held at Arcadia.
+
+| Date | Tutorial | Length | Description |
+|------|----------|--------|-------------|
+| Sept 20, 2022 | [Introduction to Git & GitHub](20220920-intro-to-git-and-github/lesson.md) | 2 hours | This workshop introduces the Git version control system and how to use GitHub for collaboration. It covers `git clone`, `git add`, `git commit`, `git push`, `git pull`, `git checkout`, `git branch`, `git status`, pull requests, and code review. |
```

Comment:
Ya I started getting really nervous that we won't have enough time to cover everything in this lesson during the workshop. 

Maybe what I'll do is write these sections so that the lesson material is all in one place, and I'll skip them if we're short on time and then sub out the AUG on GitHub & mkdocs for a session on `git reset` and `git revert`